src/Compilers/Core/Portable/Operations/OperationVisitor.cs

```diff
@@ -534,7 +534,7 @@ public virtual TResult VisitUsingStatement(IUsingStatement operation, TArgument
         {
             return DefaultVisit(operation, argument);
         }
-        
+
         public virtual TResult VisitFixedStatement(IFixedStatement operation, TArgument argument)
```

Comment:
Yeah, I have trim trailing whitespace extensions installed, but I need to remember to put that in a separate commit in the future.